ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -105,7 +105,7 @@ kern_return_t VirtualizationRoots_Init()
         return KERN_FAILURE;
     }
     
-    s_maxVirtualizationRoots = 128;
+    s_maxVirtualizationRoots = 4;
```

Comment:
Making sure the various different code paths are regularly exercised now that complexity has increased a bit.

Reply:
Comment now added

Reply:
To clarify we wouldn't want to leave this at 4 full stop right?  Was it just for a functional test run?

Reply:
No, I'm fully intending for this to stay in so that any future breakage is found before it ships to users who really do have more than 128 repos.

The resizing is so rare and has such a minuscule performance impact as to effectively be irrelevant, so I think picking a "big enough for everyone*" value here is explicitly harmful, and we *should* choose an artificially small size.